wavecrestweb/kenworthy-machine
# #KM-8: Add location section

## Description

This PR adds the location section to the Home page. ~~Currently, the content is hardcoded; I will update the component to fetch content once the CMS is set up.~~

~~Again, it seems that font size is too large on all screen sizes. I will ask Kayla if we want to scale it down.~~

### UPDATES

- Created and added ContentWrapper component to restrict the content width for screen sizes larger than 1728px
- Scaled down the font sizes
- Added conversion chart for font sizes and line heights

## Related Tickets & Documents

#KM-8 is a work in progress (dependent on CMS)

## Acceptance Criteria

- [x] The landing page has a dedicated section that explicitly states Kenworthy Machine's location and service areas.
- [x] The section is designed for easy readability, with font sizes and colors that match the overall aesthetic of the landing page.
- [x] The information provided is current, reflecting any recent changes in service areas.
- [x] Content for this section is managed through TinaCMS and fetched via an API call. 
- [x] Changes to the content in TinaCMS are reflected in the section without needing code updates.

### TO-DO:  
- [ ] Double-check if the data is correctly fetched from TinaCMS when this PR is merged 

## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
| `✓`    | :sparkles: New feature     |
|   `✓`  | 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Mobile & Desktop Screenshots/Recordings
**Large screens:**
![Screen Shot 2024-01-31 at 11 29 08 AM](https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/1928e60f-448f-4aee-bdff-6e945d984694)
**Medium screens:**
![Screen Shot 2024-01-31 at 11 29 34 AM](https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/55fbecc9-ffa9-46bf-81f1-9bb6b8e45dca)
**Small screens with the current header:**
![Screen Shot 2024-01-31 at 11 29 57 AM](https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/abc666db-570f-4a85-90b3-4ddafd14dff7)
**Small screens with the current header commented out:**
![Screen Shot 2024-01-31 at 11 30 16 AM](https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/7b2ca86f-edfa-4e39-92f1-63090c49bdf7)


## What I learned


## Testing steps
1. Pull down and check out this branch
2. Run `npm run dev`
3. Open http://localhost:3000/ in your browser of choice
4. Switch between screen sizes to see how the spacing and font size change


## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|  `✓`   | 🙅 no documentation needed    |


## What gif best describes this PR?
  ![Easy](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaG50czBqOGU3emZuejZxbGZobmFpenBieHk3OXZzcGV4Z2F4NXdmaiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/3ohhwhLRprHtEWzi1O/giphy.gif)
